ecosystem/models/configuration.py

```diff
@@ -1,20 +1,21 @@
 """Configuration for ecosystem repository."""
+from __future__ import annotations
 import json
```

Comment:
Nit: the Python convention is to have a blank line in-between `__future__` imports and normal imports.